docs/guide/getstarted/configuration.md

```diff
@@ -106,8 +106,11 @@ Default: *90000*
 ### connectionRetryCount
 Count of request retries to the Selenium server
 
+### specFileRetries
+Sets the testrunner to retry a failed spec in a suite x amount of times, the retried spec gets a new setup environment for its new test.
+
 Type: `Number`<br>
-Default: *3*
+Default: *0*
```

Comment:
Can we add back
```md
Type: `Number`<br>
Default: *3*
```

Reply:
no worries, fixed.

Reply:
@pdirt did you push it? Still see old state